src/user/style.js

```diff
@@ -0,0 +1,16 @@
+import { makeStyles } from '@material-ui/core'
+
+const title = {
+  textAlign: 'center',
+  fontSize: '2vw',
+}
+
+const paper = {
+  paddingTop: '1vh',
+  paddingBottom: '1vh',
+  textAlign: 'center',
+  fontSize: '1vw',
+}
+
+const main = { maxWidth: '80%', margin: 'auto' }
```

Comment:
Let's have a new line between the `main` and `useStyle` constants.

Reply:
and also, try to extract in separate style files for separate components.

Reply:
Done